lib/has_attachment.rb

```diff
@@ -0,0 +1,34 @@
+module HasAttachment
+  extend ActiveSupport::Concern
+
+  module ClassMethods
+    def has_attachment(field, options)
```

Comment:
Assignment Branch Condition size for has_attachment is too high. [20.2/15]<br>Rename `has_attachment` to `has_attachment?`.